lib/spack/spack/cmd/install.py

```diff
@@ -139,7 +139,25 @@ def setup_parser(subparser):
     subparser.add_argument(
         '--cdash-upload-url',
         default=None,
-        help="CDash URL where reports will be uploaded"
+        help="CDash URL where reports will be uploaded."
```

Comment:
we actually standardized on mostly lowercase and no punctuation for short help like this.  Look at the output of `spack -h`